pyproject.toml

```diff
@@ -516,4 +516,8 @@ commands = molecule {posargs:test}
 """
 
 [tool.uv]
-required-version = ">=0.5.20"
+# NOTE: When you bump the minimum uv version, you also need to change it in:
+# .pre-commit-config.yaml
+# .github/actions/install-aiida-core/action.yml
+# .readthedocs.yml
+required-version = ">=0.5.21"
```

Comment:
Sorry I did not notice this till now: Why do we put required version to 0.5.21 but bump in the other places the version to 0.5.22?

Reply:
Yes, all the fixes are in 0.5.21. We don't need to bump the minimum supported version unless really needed.